style.css

```diff
@@ -431,6 +472,76 @@ textarea {
   margin: 4rem auto 0.3rem;
 }
 
+/* style modal */
+
+.modal {
+  border: 2px solid #ebf0ee;
+  margin: 15% 7%;
+  border-radius: 10px;
+  display: none;
+  transition: 0.3s ease-in-out;
+  background-color: white;
+  position: fixed;
+  z-index: 10;
+  top: 0;
+}
+
+.modal.active {
+  display: block;
+  backdrop-filter: blur(10px);
+}
```

Comment:
All other requested changes were implemented very well 👌 Let's fix this one. First, you need a container covering the whole screen, and your modal needs to live inside this container. For this container, you need to give `background-color: rgba(0, 0, 0, 0.9)`or similar to the color on Figma template. Notice that, on Figma, there is HEX color format but we need to use rgba, so you convert it. 0.9 is opacity here and with `backdrop-filter: blur(10px);` will create the blur effect. 👍